test/gpu/interop/stream/streamChpl.chpl

```diff
@@ -0,0 +1,26 @@
+use SysCTypes;
+use SysBasic;
+use CPtr;
+
+param N = 10;
+
+extern proc vector_add(
+    a : c_ptr(c_float),
+    b : c_ptr(c_float),
+    c : c_ptr(c_float),
+    alpha : real(32),
```

Comment:
Is there a reason why alpha is declared as `real(32)` instead of `c_float`?
If not, I would change it to be `c_float` so it's consistent with the others.

Reply:
Not sure why I used real(32) there. I've changed it, thanks.